.github/workflows/test-covering-pr.yml

```diff
@@ -0,0 +1,34 @@
+name: 'Test covering my Pull Request changes'
+
+on:
+  pull_request:
+    types: [opened, reopened, synchronize]
+
+jobs:
+  tests-covering-pr:
+    runs-on: ubuntu-latest
+    strategy:
+      matrix:
+        node-version: [ 14.x ]
```

Comment:
Why using this old version of NodeJS?

Reply:
My only reason is to be consistent with the version used in the other existing GitHub actions.

Reply:
Correct. Thank you for providing the background information. The end of maintenance for Node.js 14 is 2023-04-30. Are there any plans on dropping support for it and update to a more recent version?

Reply:
Keep in mind there is a limit we can reach of minutes in GitHub actions. :)
For now, I will keep that as it is. I prefer we all use the same version.

Reply:
I created one in the naica board as a reminder: https://github.com/SUSE/spacewalk/issues/20578